### PR TITLE
[PUB-1850] Onboarding tweaks

### PR DESCRIPTION
**Pull request by anafilipadealmeida**
## Description
Hide AppShell when showing Onboarding Page.
Solve Onboarding height issue.

## Context & Notes

https://buffer.atlassian.net/browse/PUB-1850
## Checklist

-   [x] My code follows the code style and guidelines of this project. 
-   [ ] I have added tests to cover my changes.
-   [x] All new and existing tests passed.
-   [x] I have tested different user stories. 
-   [x] I have identified similar or related features and tested they are still working.
-   [x] I have performed a self-review of my own code.
-   [x] I have tested my changes/additions in the latest Chrome, Firefox, and Safari.
-   [x] I have commented my code, particularly in hard-to-understand areas.

#### Staging Deployment

To visit the URL of the staging deployment, please click "Show all checks" at the bottom of this PR and click "details" next to `bufferbotbrains/cicd-buffer-publish-legacy`

### CODE REVIEW

File: packages/onboarding/components/OnboardingManager/index.jsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ const OnboardingManager = ({
   onSkipStep,
   translations,
 }) => (
-  <div>
+  <React.Fragment>
```
Comment (karelm) [inline packages/onboarding/components/OnboardingManager/index.jsx]: does removing this div have any impact on the UI?

Comment (anafilipadealmeida) [reply packages/onboarding/components/OnboardingManager/index.jsx]: Yes, a positive impact! The div was affecting the height, which was intended to be 100%!

File: packages/profile-sidebar/middleware.js [hunk 2]
```diff
@@ -112,7 +113,8 @@ export default ({ dispatch, getState }) => next => (action) => {
           tabId: 'queue',
         })));
       // With a feature flip, temporary
-      } else if (!isPreferencePage && profiles.length === 0 && isOnBusinessTrial && hasOnboardingFeatureFlip) {
+      } else if (!isPreferencePage && isComingFromSignin
```
Comment (karelm) [inline packages/profile-sidebar/middleware.js]: this feels so horrible overall doesnt it 😂 I guess that needs must here 🤷‍♂ (no judgement on your work here @anafilipadealmeida, just a general comment about the state of this file + the approach)

File: packages/profile-sidebar/middleware.js [hunk 3]
```diff
@@ -113,11 +115,11 @@ export default ({ dispatch, getState }) => next => (action) => {
         })));
       // With a feature flip, temporary
       } else if (!isPreferencePage && profiles.length === 0 && isOnBusinessTrial && hasOnboardingFeatureFlip) {
-        dispatch(push('/new-business-trialists'));
+        dispatch(push(newBusinessTrialistsRoute));
       } else if (!isPreferencePage && profiles.length === 0 && isOnBusinessTrial) {
         dispatch(push('/new-connection-business-trialists'));
```
Comment (anafilipadealmeida) [inline packages/profile-sidebar/middleware.js]: I'm not moving this route to routers package, because this will be removed soon.

**Pull request merged**